post-docs/add-back-references.py

```diff
@@ -149,9 +154,8 @@ def generate_back_references(link, base_path):
 elif gen_type == GenerationType.providers:
     all_providers = [f.path.split("/")[-1] for f in os.scandir(docs_archive_path)
                      if f.is_dir() and "providers" in f.name]
-    for p in all_providers:
-        log.info("processing airflow provider: %s", p)
-        generate_back_references(get_github_redirects_url(p), get_provider_docs_path(p))
+    for provider in all_providers:
+        print(f"Processing airflow provider: {provider}")
+        generate_back_references(get_github_redirects_url(provider), get_provider_docs_path(provider))
 else:
-    log.error("invalid type of doc generation required. Pass one of [airflow | providers | "
-              "helm]")
+    print("[red]Invalid type of doc generation required[/]. Pass one of [airflow | providers | helm]")
```

Comment:
nit: `[red]Invalid type of doc generation requested[/]. Pass one of [airflow | providers | helm]`